qcodes/utils/dataset/doNd.py

```diff
@@ -299,13 +298,19 @@ def _handle_plotting(
     return res
 
 
-def get_plots(data: DataSet) -> Tuple[DataSet, list, list]:
+def plot_data(data: DataSet, save_pdf: bool = True,
```

Comment:
Shall we call this public function  simply `plot`? The 'data' part of the name does not give any additional information if you ask me.

Reply:
Sure, will do.